x-pack/test/security_solution_api_integration/test_suites/detections_response/basic_license/rule_management/coverage_overview.ts

```diff
@@ -0,0 +1,639 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+import {
+  RULE_MANAGEMENT_COVERAGE_OVERVIEW_URL,
+  ThreatArray,
+} from '@kbn/security-solution-plugin/common/api/detection_engine';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+import {
+  createPrebuiltRuleAssetSavedObjects,
+  createRuleAssetSavedObject,
+  createRule,
+  deleteAllRules,
+  getSimpleRule,
+  installPrebuiltRulesAndTimelines,
+  installPrebuiltRules,
+  createNonSecurityRule,
+} from '../../utils';
+
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const log = getService('log');
+  const es = getService('es');
+
+  describe('@serverless @ess coverage_overview', () => {
```

Comment:
Before merging this, test if the tests pass in QA as well, running the `npm run rule_management:qa:serverless` that you created. This however needs a little bit of setup, which @WafaaNasr described here: 
https://docs.google.com/document/d/1YUD5nD90XYZ59v1-1LW77QAnOgJrxGjKDDgNiREGu8c/edit
Reach out to me if you need help setting everything up. If the tests don't pass in QA, skip the whole suite with @skipInQa.

Reply:
Thanks @jpdjere!!
I wonder if the tests placed in the new `basic_license` will also be running in Serverless or only ESS 🤔 ?

Reply:
Hey @jpdjere @dplumlee I am sorting out the Basic/Essentials license in this [PR ](https://github.com/elastic/kibana/pull/171531), and we will get merged soon, so you don't have to worry about the licensing here anymore ^_^

Reply:
The PR got merged, and there is a new folder called `basic_essentials_license` to hold the tests that will be running in the Basic-ESS license and (or) the Essentials/Serverless